scripts/release/build-commands/update-package-versions.js

```diff
@@ -36,7 +36,13 @@ const update = async ({cwd, dry, version}) => {
       // In order to simplify DX for the release engineer,
       // These packages are auto-incremented by a minor version number.
       if (semver.lt(json.version, '1.0.0')) {
-        json.version = `0.${semver.minor(json.version) + 1}.0`;
+        const prerelease = semver.prerelease(version);
+        let suffix = '';
+        if (prerelease) {
+          suffix = `-${prerelease.join('.')}`;
+        }
+
+        json.version = `0.${semver.minor(json.version) + 1}.0${suffix}`;
```

Comment:
Not sure I understand the difference between when to use `version` and when to use `json.version`.

If I do two pre-releases in a row, does this increment minor once or twice? If it only increments once then sounds fine.

Reply:
`version` is coming from the user's CLI param (eg `16.0.1-beta`)

The "version" in `package.json` will typically be the same, except for < 1.0 releases like the reconciler, when we'll increment the major (eg `0.1.0` -> `0.2.0`) and now also append the prerelease suffix (eg `16.1.0-beta` -> `0.2.0-beta`).

Reply:
I'll update the inline comment to be clearer.